src/sql/azdata.proposed.d.ts

```diff
@@ -670,10 +670,36 @@ declare module 'azdata' {
 		 */
 		delete?: boolean;
 	}
+
 	export interface DiffEditorComponent {
 		/**
 		 * Title of editor
 		 */
 		title: string;
 	}
+
+	export interface TextColumnOption {
+		content?: string
+	}
+
+	export interface RowDetailsOptions {
+		panelRows: number,
+		detailsHtml: boolean,
+		headerCssClass?: string,
```

Comment:
How are callers supposed to know what classes to use? Generally the pattern we use is that callers can specify the css styles to apply themselves so I'd rather you do that here too.

Reply:
This is done the same way like columns do. 
{
value: 'Severity',
field: 'severity',
headerCssClass: 'no-borders align-with-header',
width: 100
}

RowDetails is actually a column, so we need to specify its css class like regular column.

Reply:
Gotcha - well not my favorite pattern but guess better to stay consistent in this case